tests/unit/assessment/test_aws.py

```diff
@@ -336,6 +585,86 @@ def upload(
     aws_resource_permissions.save_instance_profile_permissions()
 
 
+def test_save_uc_compatible_roles():
+    ws = create_autospec(WorkspaceClient)
+    ws.current_user.me = lambda: iam.User(user_name="me@example.com", groups=[iam.ComplexValue(display="admins")])
+
+    expected_csv_entries = [
+        "arn:aws:iam::12345:role/uc-role1,s3,READ_FILES,s3://bucket1",
+        "arn:aws:iam::12345:role/uc-role1,s3,READ_FILES,s3://bucket2",
+        "arn:aws:iam::12345:role/uc-role1,s3,READ_FILES,s3://bucket3",
+        "arn:aws:iam::12345:role/uc-role1,s3,WRITE_FILES,s3://bucketA",
+        "arn:aws:iam::12345:role/uc-role1,s3,WRITE_FILES,s3://bucketB",
+        "arn:aws:iam::12345:role/uc-role1,s3,WRITE_FILES,s3://bucketC",
+    ]
+
+    def upload(
```

Comment:
use `MockInstallation` for this

see https://github.com/databrickslabs/blueprint?tab=readme-ov-file#application-and-installation-state